### DISCUSSION TITLE
Double paypal buttons issue

### DISCUSSION THREAD

**Discussion by virajsoni06** [swellstores/community#352] [Help]
Hi, we are using hosted swell checkout and there seems to be a bug here. The Paypal button appears twice at times
![image](https://github.com/swellstores/community/assets/13236286/eae12b76-470c-4e8b-b08f-749ebfb42ced)

See the image above, its quite random though it does appear quite often.

**Comment by SusannaSwell** [comment]
@virajsoni06 hello!  we have this bug filed in our internal system, once we have an update, I will get back to you.

**Comment by steven-k1** [comment]
@virajsoni06 We now have our new PayPal Commerce integration available, which is a replacement for the older PayPal Express one you are using and experiencing this issue with.

You can read up on it [here](https://www.swell.is/blog/new-paypal-integration) if you would like, but it has the same functionality as the PayPal Express one and some added functionality that the older integration doesn't offer.

Switching to the use of this integration would be a quick workaround for you.

**Comment by virajsoni06** [comment]
![image](https://github.com/swellstores/community/assets/13236286/e2d26bcc-857c-4519-b381-8f9b6e27040c)

Hi @steven-k1, it says we won't have access to legacy payment options. That's kind of restrictive, we would like to have access to existing PayPal Express, if PayPal Commerce isn't right for us, we want to change it back to PayPal Express

**Reply by steven-k1** [reply]
@virajsoni06 The newer PayPal Commerce integration we offer has the same features as the PayPal Express integration, along with added functionality. It was created to replace the outdated PayPal Express integration, which is why you see this message.
